doc/examples/applications/plot_3d_image_processing.py

```diff
@@ -55,7 +55,7 @@
 # Load and display 3D images
 # ==========================
 
-data = ski.util.img_as_float(ski.data.cells3d()[:, 1, :, :])  # grab just the nuclei
+data = ski.util.rescale_to_float(ski.data.cells3d()[:, 1, :, :])  # grab just the nuclei
```

Comment:
In places where `rescale_to_float*` functions are seen by users, I checked whether setting `legacy_float_range=True` is necessary. In this case, `cells3d()` returns a an uint8-array. So this parameter has no effect (it's only meaningful for signed ints).